python/lsst/obs/base/formatters/fitsExposure.py

```diff
@@ -418,21 +422,94 @@ def checkUnrecognized(entry, allowed, description):
         return validated
 
 
-class FitsImageFormatter(FitsExposureFormatter):
+class FitsExposureFormatter(FitsImageFormatterBase):
+    """Specialization for `~lsst.afw.image.Exposure` reading.
+    """
+
+    _readerClass = ExposureFitsReader
+
+    def _compareFilters(self, file_filter_label):
```

Comment:
:+1: to `_fixFilterLabel`.  I didn't go with that originally because the originally logic _didn't_ fix it most of the time, but now it makes more sense.